README.md

```diff
@@ -1,8 +1,17 @@
-# React + Vite
+# ToDo APP
+## Brief Description
+This repository contains the source code and documentation for the creation of the To-Do List App. This app is designed to help users manage and organize their daily tasks in an efficient and effective way.
 
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+### how to contribute
+1. Open terminal and clone this repository to your machine local using `git clone repositorylink`.
+2. go to the project directory wtih `cd repo-name`.
+3. Install the dependencies by using Yarn `yarn install
+`.
+4. Run the application with the Yarn command `yarn start`.
```

Comment:
@Azissa do we have this `start` command in `scripts` section of `package.json` file?

Reply:
no sir,do i have to delete the no.4 and replace it with something like `git branch` and `git push`?

Reply:
@Azissa no. I mean, use one of command listed in that section. the command should be `yarn dev`

Reply:
okay got it sir